web/src/buddy_list.js

```diff
@@ -54,12 +60,26 @@ class BuddyListConf {
     }
 }
 
+function would_receive_message(user_id) {
+    const current_sub = narrow_state.stream_sub();
+    if (current_sub) {
+        return stream_data.is_user_subscribed(current_sub.stream_id, user_id);
+    }
+    const pm_ids_string = narrow_state.pm_ids_string();
+    if (pm_ids_string) {
+        const ids = pm_ids_string.split(",");
+        return ids.includes(user_id.toString());
+    }
+    return false;
+}
+
 export class BuddyList extends BuddyListConf {
     keys = [];
 
     populate(opts) {
         this.render_count = 0;
-        this.$container.empty();
+        this.$narrow_users_container.empty();
```

Comment:
Similarly with a name like `matching_users_container`?

Reply:
Yeah I was not really sure what to do with naming for some of these things, so this is helpful :)